armi/resources/referenceFissionProducts.dat

```diff
@@ -145,43 +134,30 @@
 13          LFP35 EU1547 2.0000E-04
 13          LFP35 EU1557 3.4000E-04
 13          LFP35 EU1567 2.0000E-05
-13          LFP35 GD1547 0.0000E+00
 13          LFP35 GD1557 1.0000E-05
 13          LFP35 GD1567 1.7000E-04
 13          LFP35 GD1577 7.0000E-05
 13          LFP35 GD1587 6.0000E-05
 13          LFP35 GD1607 1.0000E-05
 13          LFP35 TB1597 3.0000E-05
-13          LFP35 TB1607 0.0000E+00
-13          LFP35 TB1617 0.0000E+00
-13          LFP35 DY1607 0.0000E+00
-13          LFP35 DY1617 0.0000E+00
-13          LFP35 DY1627 0.0000E+00
-13          LFP35 DY1637 0.0000E+00
-13          LFP35 DY1647 0.0000E+00
```

Comment:
Removed these because they were zero and had no impact and removed Y90 because its not a nuclide that is modeled in MC2-3.